style.css

```diff
@@ -0,0 +1,219 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family:'Roboto', sans-serif;;
+}
+
+.parent {
+    width: 100%;
+}
+
+.child {
+    width: 80%;
+    margin: auto;
+}
+.container1{
+    background-color: #EEEEEE;
+    padding: 4%;
+    border: 1px solid #E2E2E2;
+}
+/* Sample Heading */
+.container1 h1{
```

Comment:
avoid targeting through tag names.
It may affect other elements too inside that div.
use class name as a first preference